index.template.html

```diff
@@ -4715,16 +4432,17 @@ <h6>Content Types</h6>
             <h6>Data Schemas</h6>
 
             <p>
-              Data Schema, as explained in [[WOT-THING-DESCRIPTION]], describes the structure of the messages, which are
-              used together with media types. Even though it is largely inspired by JSON Schema [[json-schema]], it can
-              be used for describing other payload types such as [[XML]], string-encoded images, bit representations of
-              integers, etc. Data Schema SHOULD be used in addition to the media types.
+              Data Schema, as explained in [[#sec-data-schema-vocabulary-definition"]], describes the structure of the
```

Comment:
Yet again, [[#sec-data-schema-vocabulary-definition"]] does not link to anything.